docs/eventing/samples/kafka/channel/README.md

```diff
@@ -37,7 +37,7 @@ EOF
 ## Specifying the default channel configuration
 
 To configure the usage of the `KafkaChannel` CRD as the
-[default channel configuration](../../../channels/default-channels.md), edit the
+[default channel configuration](../../../channels/channel-types-defaults.md), edit the
```

Comment:
```suggestion
[default channel configuration](../../../channels/channel-types-defaults), edit the
```

Reply:
I removed the MD suffix from this link, but now it appears invalid in my editor.  I assume that in final docs, a link to an MD file is valid without the file extension?

Reply:
Ah I see in the deployed preview that this does in fact work